include/tests_crypto

```diff
@@ -223,6 +223,29 @@
     fi
 #
 #################################################################################
+#
+    # Test        : CRYP-8005
+    # Description : Test for presence of software pseudo random number generators
+    Register --test-no CRYP-8005 --os Linux --weight L --network NO --root-only NO --category security --description "Presence of software pseudo random number generators"
+    if [ ${SKIPTEST} -eq 0 ]; then
+        LogText "Test: looking for software pseudo random number generators"
+	FOUND=""
+	for SERVICE in audio-entropyd havegd jitterentropy-rngd; do
+            if IsRunning "${SERVICE}" --full; then
+		FOUND="${FOUND} ${SERVICE}"
+		echo found $FOUND
```

Comment:
Suggestion: remove this line or use LogText to add an entry to the log file.

Reply:
Sorry, that's just debugging which I forgot to remove.